src/Mod/Draft/App/AppDraftUtilsPy.cpp

```diff
@@ -56,38 +51,8 @@ class Module : public Py::ExtensionModule<Module>
 private:
     Py::Object readDXF(const Py::Tuple& args)
     {
-        Base::Console().Warning("DraftUtils.readDXF is deprecated. Use Import.readDxf instead.\n");
-        char* Name;
-        const char* DocName=nullptr;
-        bool IgnoreErrors=true;
-        if (!PyArg_ParseTuple(args.ptr(), "et|sb","utf-8",&Name,&DocName,&IgnoreErrors))
-            throw Py::Exception();
-
-        std::string EncodedName = std::string(Name);
-        PyMem_Free(Name);
-
-        Base::FileInfo file(EncodedName.c_str());
-        if (!file.exists())
-            throw Py::RuntimeError("File doesn't exist");
-
-        App::Document *pcDoc;
-        if (DocName)
-            pcDoc = App::GetApplication().getDocument(DocName);
-        else
-            pcDoc = App::GetApplication().getActiveDocument();
-        if (!pcDoc) 
-            pcDoc = App::GetApplication().newDocument(DocName);
-
-        try {
-            // read the DXF file
-            DraftDxfRead dxf_file(EncodedName,pcDoc);
-            dxf_file.DoRead(IgnoreErrors);
-            pcDoc->recompute();
-        }
-        catch (const Base::Exception& e) {
-            throw Py::RuntimeError(e.what());
-        }
-
+        Base::Console().Warning("DraftUtils.readDXF is removed. "
```

Comment:
I don't think it's used anywhere anymore in the FreeCAD codebase itself. Maybe in external addons... But since all this DraftUtils C++ module is there only for DXF, I think it's a good move, so addon authors make the switch, and later on we can remove the C++ stuff entirely and get Draft back to being a pure Python module.